### PR TITLE
add perf permission for parallel-rustc wg

### PR DESCRIPTION
**Pull request by SparrowLii**
The work of the parallel compiler is very dependent on the perf tool, I hope to obtain the permission to use the tool

### CODE REVIEW

File: teams/wg-parallel-rustc.toml [hunk 1]
```diff
@@ -19,6 +19,9 @@ alumni = [
     "spastorino",
 ]
 
+[permissions]
+perf = true
+
```
Comment (lqd) [inline teams/wg-parallel-rustc.toml:24]: Unless the `perf` permission also implies the right to trigger `try` builds (I don't believe it's the case), I think the following permission is also going to be needed.

```toml
bors.rust.try = true
```

**Comment by lqd** [comment]
(IIRC the team's lead approval will also be needed, therefore cc-ing wg-parallel-rustc's lead @cjgillot)

**Comment by cjgillot** [comment]
Ok on my end.

File: teams/wg-parallel-rustc.toml (hunk 1 above)
Comment (SparrowLii) [reply teams/wg-parallel-rustc.toml:25]: Thanks for pointing out! I have added the line

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**